crates/oxc_query/src/edges.rs

```diff
@@ -1772,19 +1774,15 @@ mod return_statement_ast {
         _resolve_info: &ResolveEdgeInfo,
     ) -> ContextOutcomeIterator<'a, Vertex<'b>, VertexIterator<'a, Vertex<'b>>> {
         resolve_neighbors_with(contexts, |v| {
-            #[allow(clippy::option_if_let_else)]
-            if let Some(expr) = &v
+            let neighbors = v
                 .as_return_statement_ast()
-                .unwrap_or_else(|| {
-                    panic!("expected to have a returnstatementast vertex, instead have: {v:#?}")
-                })
+                .unwrap()
```

Comment:
`.unwrap()` -> `.expect()` again

Reply:
The point of the or_else is because panic uses a format string with the v arg, so it's preferable to not do that in the .expect which is always executed

Reply:
when I was reviewing this was just `.unwrap()` not `.unwrap_or_else()`. This is fine.

Reply:
Btw as far as I can tell this is still `.unwrap()` not `.unwrap_or_else()`. Can you double-check this in the actual PR code viewer, not in the summary page of the PR? They don't seem to be in sync.

Reply:
it's now .into_iter